### ISSUE TITLE
Test trained model in different dataset

### ISSUE THREAD

**Issue by yutongy98** [hasanirtiza/Pedestron#132]
Hi, I trained the HTC model on the training set of the cityperson dataset, and now I want to evaluated it on the validation set of the eurocity persons dataset to have a look on the generalization of the model. Is it possible to do so? Thanks.

**Comment by hasanirtiza** [comment]
Of course, yes. Please have a look at the readme.md file on how to test on the ECP. All you have to do is provide the path. This command should work `python ./tools/test_euroCity.py configs/elephant/eurocity/cascade_hrnet.py ./models_pretrained/epoch_ 1 2 --mean_teacher
`

for `models_pretrained` provide you model folder.
